### ISSUE TITLE
Let's improve the accessibility of the website

### ISSUE THREAD

**Issue by tryzniak** [webpack/webpack.js.org#2244]
Hello. I ran Google Chrome's audit on accessibility and it turns out there are a number of issues. A lot of them it looks to me can be easily fixed like missing `alt` attributes, buttons without a name, low contrast. 
I can work on a PR btw :)

**Comment by montogeek** [comment]
Send them :)

**Comment by jeffin143** [comment]
![IMG_20191206_010530](https://user-images.githubusercontent.com/21286134/70267462-845df380-17c4-11ea-94b1-904ab413bb03.jpg)


It has reached 99 :)

Loved it , let's close this 
May be we could hit 100 some day :-p
